swap/swap.go

```diff
@@ -130,7 +131,7 @@ func swapRotatingFileHandler(logdir string) (log.Handler, error) {
 }
 
 // newSwapInstance is a swap constructor function without integrity checks
-func newSwapInstance(stateStore state.Store, owner *Owner, backend contract.Backend, params *Params, chequebookFactory contract.SimpleSwapFactory) *Swap {
+func newSwapInstance(stateStore state.Store, owner *Owner, backend contract.Backend, chainID uint64, params *Params, chequebookFactory contract.SimpleSwapFactory) *Swap {
```

Comment:
It would be great to have the order of function arguments the same as the order of the functions in the swap struct

Reply:
This already wasn't the case prior to the PR

Reply:
That being said I want the backend and chain id next to each other since they kind of belong together. So it should be changed in the struct.